### ISSUE TITLE
[readability-misleading-indentation] False report for macro statement

### ISSUE THREAD

**Issue by timxx** [llvm/llvm-project#71767]
False report when the statement starts with an empty macro, it's common case in Qt's signal/slot.

```C++
if (condition)
    foo();
else if (another)
    bar();
emit theSignal();
```

The `emit` is `# define emit`
